index.css

```diff
@@ -0,0 +1,122 @@
+@import url("https://vercel-ui-preview.netlify.app/component-lib/all-comp.css");
+@import url("https://vercel-ui-preview.netlify.app/component-lib/all-comp.css");
+* {
+	margin: 0;
+	box-sizing: border-box;
+	font-size: 100%;
+}
+html {
+	font-size: 62.5%;
+}
+main {
+	min-height: 100vh;
+	font-size: 1.6rem;
+	padding: 0 1.6rem;
+	display: grid;
+	gap: 1.6rem;
+	grid-template-rows: auto 1fr 1fr 1fr auto;
+}
+
+.header {
+	width: 100%;
+	display: flex;
+	align-items: center;
+	justify-content: space-between;
+	box-shadow: 0 5px 3px -3px var(--box-shadow);
+}
+
+.header-sub-container {
+	display: flex;
+	align-items: center;
+	gap: 0 2rem;
+}
+.btn {
+	color: #2d3748;
```

Comment:
you can use variable for color

Reply:
Implemented suggestion in the latest commit.